eng/test-job.yml

```diff
@@ -203,6 +203,35 @@ jobs:
           - jitstress2_tiered
           - zapdisable
           - tailcallstress
+        ${{ if eq(parameters.testGroup, 'outerloop-jitstress-isas-arm') }}:
+          scenarios:
+          - jitstress_isas_incompletehwintrinsic
```

Comment:
The first four jobs are shared between both x86 and ARM and so they don't have an architecture moniker in the name.

They were duplicated in each listing.

Reply:
This list will expand quite a bit once the work on the ARM64 HWIntrinsics starts up. But for now, it should be "good enough" for the experimental HWIntrinsics we currently have and the pre-existing `System.Numerics.Vector` support.